### PR TITLE
[DO-1770] Create PEP27 scoring rule

### PR DESCRIPTION
**Pull request by Andrii17s**
PEP27

PEP declared rent of real estate exceeding 300 sq. m.

### CODE REVIEW

File: business_register/pep_scoring/rules.py [hunk 1]
```diff
@@ -180,3 +180,34 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+
+    rule_id = ScoringRuleEnum.PEP27
+
+    class DataSerializer(serializers.Serializer):
+        square_meters = serializers.IntegerField(min_value=0, required=True)
+
+    def calculate_weight(self) -> Tuple[Union[int, float], dict]:
+        square_meters = 0
+        property_types = [Property.SUMMER_HOUSE, Property.HOUSE, Property.APARTMENT, Property.ROOM,
+                          Property.GARAGE, Property.UNFINISHED_CONSTRUCTION, Property.OTHER, Property.OFFICE]
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:200]: I think we should discuss with ANTAC do we need OTHER and OFFICE types here

File: business_register/pep_scoring/rules.py [hunk 2]
```diff
@@ -180,3 +180,34 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+        for property in PropertyRight.objects.filter(
+            property__declaration_id=self.declaration.id,
+            property__type__in=property_types,
+            type=PropertyRight.RENT,
+        ).values_list('property__area', flat=True)[::1]:
+            square_meters += property
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:206]: I am not sure we can sum here

File: business_register/pep_scoring/rules.py [hunk 3]
```diff
@@ -180,3 +180,34 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+    rule_id = ScoringRuleEnum.PEP27
+
+    class DataSerializer(serializers.Serializer):
+        square_meters = serializers.IntegerField(min_value=0, required=True)
+
+    def calculate_weight(self) -> Tuple[Union[int, float], dict]:
+        square_meters = 0
+        property_types = [Property.SUMMER_HOUSE, Property.HOUSE, Property.APARTMENT, Property.ROOM,
+                          Property.GARAGE, Property.UNFINISHED_CONSTRUCTION, Property.OTHER, Property.OFFICE]
+        for property in PropertyRight.objects.filter(
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:201]: for `property_area` would be more clear, I guess

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/pep_scoring/rules.py [hunk 4]
```diff
@@ -253,3 +253,35 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+                type=PropertyRight.RENT,
+        ).values_list('property__area', flat=True)[::1]:
+            try:
+                if property_area > 300:
+                    weight = 0.3
+                    data = {
+                    "square_meters": property_area,
+                    }
+                    return weight, data
+            except:
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:285]: except all errors its bad practice

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/pep_scoring/rules.py [hunk 5]
```diff
@@ -253,3 +253,33 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+    """
+    Rule 27 - PEP27
+    weight - 0.3
+    PEP declared rent of real estate exceeding 300 sq. m.
+    """
+
+    rule_id = ScoringRuleEnum.PEP27
+
+    class DataSerializer(serializers.Serializer):
+        square_meters = serializers.IntegerField(min_value=0, required=True)
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:269]:
```suggestion
        square_meters = serializers.FloatField(min_value=0, required=True)
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/pep_scoring/rules.py [hunk 6]
```diff
@@ -253,3 +253,36 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+    """
+    Rule 27 - PEP27
+    weight - 0.3
+    PEP declared rent of real estate exceeding 300 sq. m.
+    """
+
+    rule_id = ScoringRuleEnum.PEP27
+    message_uk = ('кількість об’єктів орендованої жилої нерухомості, що перевищують '
+                  'площу 300 м. кв. {bigger_area_counter}')
+    message_en = 'amount of rented real estate exceeding 300 sq.m. {bigger_area_counter}',
```
Comment (roman-tiukh) [inline business_register/pep_scoring/rules.py:269]:
```suggestion
    message_en = 'amount of rented real estate exceeding 300 sq.m. {bigger_area_counter}'
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**